Sources/Kommunicate/Classes/Kommunicate.swift

```diff
@@ -357,6 +357,29 @@ open class Kommunicate: NSObject,Localizable{
             }
         })
     }
+    
+    /**
+     Updates the conversation parameters.
```

Comment:
Inform that they have to pass conversation ID and any other conversation property that they want to update.